src/olympia/reviews/serializers.py

```diff
@@ -96,7 +98,7 @@ def validate(self, data):
             # Only one level of replying is allowed, so if it's already a
             # reply, we shouldn't allow that.
             raise serializers.ValidationError(
-                'Can not reply to a review that is already a reply.')
+                _(u"You can't reply to a review that is already a reply."))
```

Comment:
Yeah, it's no biggie. Your original message works. I agree that it probably won't ever be shown to the user (if so, it will be a UI bug).